packages/core/src/_typography.scss

```diff
@@ -190,10 +190,11 @@ Markup:
 <pre>
 %pt-select {
   @include pt-button();
-  @include prefixer(appearance, none, webkit moz);
   border-radius: $pt-border-radius;
   height: $pt-button-height;
   padding: 0 ($input-padding-horizontal * 3) 0 $input-padding-horizontal;
+  -moz-appearance: none;
```

Comment:
ooc, none of our tooling does automatic prefixing by default?

Reply:
oof actually no, we don't run autoprefixer on the generated CSS file. one of the things we lost in the (gulp) fire, along with comment stripping.